.github/workflows/build.yml

```diff
@@ -22,6 +22,7 @@ jobs:
     steps:
       - run: dnf install --nogpgcheck -y make git-core rpm-build 'dnf-command(builddep)'
       - uses: actions/checkout@v2
+      - run: git config --global --add safe.directory /__w/selinux-policy/selinux-policy
```

Comment:
What about `chown -R $(id -u):$(id -g) .`?

Reply:
Well, this was one of the options while the PR was in a draft state. Now also the commit messages has been elaborated on, so it is ready for review.